### ISSUE TITLE
got this error after a while

### ISSUE THREAD

**Issue by mafintosh** [mappum/electron-webrtc#41]
```
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: Error evaluating "
        (function () {
          var id = "1l"
          var reqId = "f91043b8add851ca16edb170a6683cce"
          var pc = conns[id]
          var onSuccess = function (res) {
            send(reqId, { res: res && res.toJSON ? res.toJSON() : res })
          }
          var onFailure = function (err) {
            send(reqId, { err: err })
          }
          pc.close()
        })()
      ": Failed to execute 'close' on 'RTCPeerConnection': The RTCPeerConnection's signalingState is 'closed'. in "[\"3ya\",{\"err\":\"Failed to execute 'close' on 'RTCPeerConnection': The RTCPeerConnection's signalingState is 'closed'.\"}]\n"
    at Parser._transform (/Users/maf/dev/node_modules/hyperirc/node_modules/electron-webrtc/node_modules/electron-eval/node_modules/newline-json/parser.js:30:12)
    at Parser.Transform._read (_stream_transform.js:167:10)
    at Parser.Transform._write (_stream_transform.js:155:12)
    at doWrite (_stream_writable.js:301:12)
    at writeOrBuffer (_stream_writable.js:287:5)
    at Parser.Writable.write (_stream_writable.js:215:11)
    at Socket.ondata (_stream_readable.js:536:20)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:153:18)
```

**Comment by mafintosh** [comment]
seems to happen when i close my laptop so maybe this is related to a disconnect?

**Comment by mappum** [comment]
Thanks for reporting this! Fixed as of version 0.2.6.
